ftw/contacts/profiles/simplelayout/types/ftw.contacts.Member.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0"?>
+<object name="ftw.contacts.MemberBlock"
+        meta_type="Dexterity FTI"
+        xmlns:i18n="http://xml.zope.org/namespaces/i18n"
+        i18n:domain="ftw.contacts" >
+
+    <!-- add permission -->
+    <property name="add_permission">ftw.contacts.AddMemberBlock</property>
+
```

Comment:
this FTI definition looks quite empty, why is it missing all the other things? :open_mouth: